extras/videoDrivers/SDL2/VideoSDL2.cpp

```diff
@@ -223,6 +223,12 @@ void VideoSDL2::PrintError(const std::string& msg) const
     boost::nowide::cerr << msg << std::endl;
 }
 
+void VideoSDL2::ShowErrorMessage(const std::string& title, const std::string& message)
+{
+    if(window)
+        SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, title.c_str(), message.c_str(), window);
```

Comment:
```suggestion
    SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, title.c_str(), message.c_str(), window);
```

"the parent window, or NULL for no parent" -> NULL is fine, better than ignoring this.

Reply:
done